### ISSUE TITLE
can't load camera on desktop browser. 

### ISSUE THREAD

**Issue by trisutrisno-id** [WAGFS/vue-zxingjs#5]
vue-zxing.umd.js?3468:36766 

Uncaught (in promise) OverconstrainedError {name: "OverconstrainedError", message: "", constraint: "facingMode"}

**Comment by WAGFS** [comment]
Ok, I think this is because the computer has a camera but can only call the front camera, and the vue-zxing plug-in detects that the device has a camera and calls the device's rear camera by default. I will take the time to fix this problem. The original intention of making this plugin was mainly for mobile devices😋

**Comment by trisutrisno-id** [comment]
yes, i aware to it. i'm testing with @zingjs/library only and it's work. 

hope you can fix it.

**Comment by WAGFS** [comment]
Okay, I will take the time to fix it. Thank you for your questions. If you are particularly anxious, you can go to @zingjs/library to [**check this kind of problem**](https://github.com/zxing-js/library/issues/122 )first, hoping to help you.

**Comment by trisutrisno-id** [comment]
i have tested your wrapper on mobile and it works. but it would be better if it support for desktop

**Comment by WAGFS** [comment]
Haha, I am busy at the moment, I will fix this bug as soon as I have time.🥰

**Comment by trisutrisno-id** [comment]
ok. take your time. 

i'm gonna create wrapper for my self for right now. 

cause this not practical yet for my project.

but i'm gonna look your for reference.

thanks..

**Comment by WAGFS** [comment]
You are welcome, I am glad to be able to help you.
